lib/ansible/modules/remote_management/intersight/intersight_facts.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: intersight_facts
+short_description: Gather facts about Intersight
+description:
+- Gathers facts about servers in L(Cisco Intersight,https://intersight.com).
+extends_documentation_fragment: intersight
+options:
+  server_names:
+    description:
+    - Server names to retrieve facts from.  An empty list will return all servers.
```

Comment:
Please make this two items in the description.
```suggestion
    - Server names to retrieve facts from.
    - An empty list will return all servers.
```